Makefile

```diff
@@ -1,16 +1,23 @@
-.PHONY: ide stylish-haskell hlint
```

Comment:
adding `.PHONY` above each target is more typing but it's far easier to maintain.